webapp/components/test/wpt-app.html

```diff
@@ -18,16 +19,43 @@
 import '../../views/wpt-app.js';
 import { TEST_RUNS_DATA } from './util/helpers.js';
 
+suiteSetup(() => {
+  window.fetch = (url) => {
+    const href = url instanceof URL ? url.href : 'unknown';
+    assert.fail('actual', 'expected', `uncaptured fetch: ${href}`);
+  };
+});
+
 suite('<wpt-app>', () => {
-  let appFixture = null;
+  let sandbox;
 
   setup(() => {
-    appFixture = fixture('wpt-app-fixture');
-    appFixture.path = '/';
+    sandbox = sinon.sandbox.create();
+    // Spoof an empty result for APIs to speed up tests.
+    // This is done in suiteSetup/suiteTeardown to avoid async fetches slipping
+    // through between tests.
+    const ignore = new RegExp('/api/(shas|versions|interop)');
+    sandbox.stub(window, 'fetch', url => {
+      if (url === undefined) {
```

Comment:
This is likely fine, but if we didn't check this wouldn't we just throw anyway when we accessed `url.pathname` (which is also fine)?

Reply:
I'm leaving this as is for now.